src/main/steps/common/task-list/content.ts

```diff
@@ -140,6 +140,7 @@ export const generateContent: TranslationFn = content => {
   const caseData = request.session.userCase;
   const partyType = request.params.partyType;
   const _isRepresentedBySolicotor = isRepresentedBySolicotor(caseData, request.session.user.id);
+  const hideTraintrack = false;
```

Comment:
Set up a feature toggle in LD to control.

Reply:
done